package.json

```diff
@@ -1111,7 +1126,7 @@
             "args": [],
             "cwd": "${workspaceRoot}",
             "stopAtEntry": false,
-            "externalConsole": false
+            "console": "internalConsole"
```

Comment:
I discussed this with Charlie offline. The problem is that the terminal will get hidden by the debug console by default, so it may not be very obvious where to go and interact with the application. So we will leave it this way for now.